### ISSUE TITLE
Prioritizing neoepitopes

### ISSUE THREAD

**Issue by tarini92** [pdxgx/neoepiscope#6]
Hi,

After getting the neoepitopes, I thought of a way to prioritize by using the availability of the allele and the binding score of the peptide with that allele as a factor. As you mentioned, you have ranked the alleles based on their normalized presence in the population, what tools did you use to calculate the allele frequence population wise, and also,can that data be made available?

**Comment by maryawood** [comment]
Hi,

This isn't something that we have incorporated into `neoepiscope` at present, but we did do an analysis incorporating population allele frequencies into another project described in this manuscript: https://bmccancer.biomedcentral.com/articles/10.1186/s12885-018-4325-6

Is this the analysis you're referring to? If so, we used the allele frequency values from the Allele Frequency Net Database, as summarized by the HLA typing tool POLYSOLVER. Hope this helps!

**Comment by tarini92** [comment]
Yes, this is the paper that I was referring. Thanks. I’ll look up the Allele Frequency Net DB.
